### PR TITLE
Add description of ZRTP

### LINKED ISSUE #8: Protocol: ZRTP (subsection of SRTP)

### PR DESCRIPTION
**Pull request by csperkins**
Closes #8

### CODE REVIEW

File: draft-ietf-taps-transport-security.md [hunk 1]
```diff
@@ -517,9 +518,26 @@ WebRTC) channel.
...
+Key agreement is performed using a Diffie-Hellman key exchange that runs
+on the media path. This generates a shared secret that is then used to
+generate the master key and salt for SRTP.
+
+ZRTP does not rely on a PKI or external identity management system.
+Rather, it uses an ephemeral Diffie-Hellman key exchange with hash
+commitment to allow detection of man-in-the-middle attacks.
+This requires endpoints to display a short authentication string that the
+users must read and verbally compare in order to validate the hashes.
+Key continuity is used, so this comparison need only be performed once for
```
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]: Can you give a description of key continuity?

File: draft-ietf-taps-transport-security.md [hunk 2]
```diff
@@ -517,9 +518,26 @@ WebRTC) channel.
...
+a given user.
+ZRTP avoids some complexity of signalling and key management when compared
```
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]: Can you give examples of this complexity that's avoided (aside from PKI shenanigans)? If there is nothing else, perhaps we should remove this sentence. It's redundant with the information above.

File: draft-ietf-taps-transport-security.md (hunk 1 above)
Comment (csperkins) [reply draft-ietf-taps-transport-security.md]: Will changes to say "Endpoints cache some key material after the first call to use in subsequent calls, this is mixed in with the Diffie-Hellman shared secret, so the short authentication string need only be checked once for a given user. This gives key continuity properties analogous to the secure shell (ssh)", which is a paraphrase of the ZRTP spec.

File: draft-ietf-taps-transport-security.md (hunk 2 above)
Comment (csperkins) [reply draft-ietf-taps-transport-security.md]: It's just the PKI stuff, so will remove.

File: draft-ietf-taps-transport-security.md (hunk 1 above)
Comment (chris-wood) [reply draft-ietf-taps-transport-security.md]: Works for me. :)

File: draft-ietf-taps-transport-security.md (hunk 2 above)
Comment (chris-wood) [reply draft-ietf-taps-transport-security.md]: 👍

File: draft-ietf-taps-transport-security.md [hunk 3]
```diff
@@ -517,9 +518,25 @@ WebRTC) channel.
...
+
+ZRTP does not rely on a PKI or external identity management system.
+Rather, it uses an ephemeral Diffie-Hellman key exchange with hash
+commitment to allow detection of man-in-the-middle attacks.
+This requires endpoints to display a short authentication string that the
+users must read and verbally compare to validate the hashes and ensure security.
+Endpoints cache some key material after the first call to use in subsequent
+calls; this is mixed in with the Diffie-Hellman shared secret, so the short
+authentication string need only be checked once for a given user.  This
+gives key continuity properties analogous to the secure shell (ssh).
```
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]: Should we cite SSH?

**Review by chris-wood — APPROVED** [review]

File: draft-ietf-taps-transport-security.md (hunk 3 above)
Comment (csperkins) [reply draft-ietf-taps-transport-security.md]: Is there a spec to cite?

Comment (chris-wood) [reply draft-ietf-taps-transport-security.md]: RFC4253, maybe?

Comment (csperkins) [reply draft-ietf-taps-transport-security.md]: works for me

**Pull request merged**